types/packages/caver-contract/src/index.d.ts

```diff
@@ -0,0 +1,126 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import BN from 'bn.js'
+import { IWallet } from '../../caver-wallet/src'
+import { Result } from '../../caver-abi/src'
+import { KIP37 } from '../../caver-kct/src'
+import KIP7 from '../../caver-kct/src/kip7'
+import { TransactionReceipt, CallObject, BlockNumber, PromiEvent, LogsOptions } from '../../caver-core/src'
+import { Transaction, FeeDelegatedTransaction } from '../../caver-transaction/src'
+import { AbiItem } from '../../caver-utils/src'
+import KeyringContainer from '../../caver-wallet/src'
+
+export interface SendOptions {
+    from?: string
+    gasPrice?: number | string
+    gas?: number | string
+    data?: string
```

Comment:
This field don't define in SendOptions.

Reply:
This can be used for deploy.
In original logic, `deploy` can be used without `data` if default options of the contract has `data`.